examples/sum.rs

```diff
@@ -8,12 +8,12 @@ use prio::server::*;
 fn main() {
     let priv_key1 = PrivateKey::from_base64(
         "BIl6j+J6dYttxALdjISDv6ZI4/VWVEhUzaS05LgrsfswmbLOgN\
-        t9HUC2E0w+9RqZx3XMkdEHBHfNuCSMpOwofVSq3TfyKwn0NrftKisKKVSaTOt5seJ67P5QL4hxgPWvxw==",
+         t9HUC2E0w+9RqZx3XMkdEHBHfNuCSMpOwofVSq3TfyKwn0NrftKisKKVSaTOt5seJ67P5QL4hxgPWvxw==",
```

Comment:
What accounts for this extra space?

Reply:
This is just the output of `cargo fmt` on my machine. I wonder if this is different between versions of rust? As for the leading whitespace, it's ignored as per https://doc.rust-lang.org/reference/tokens.html#string-literals.

Reply:
Ah interesting. I guess that's why `cargo fmt` feels comfortable modifying this string :)